pulp_smash/tests/pulp3/file/api_v3/test_repo_version.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+"""Tests related to repository version."""
+import unittest
+from random import choice
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import FILE_FEED_URL
+from pulp_smash.tests.pulp3.constants import (
+    FILE_CONTENT,
+    FILE_IMPORTER_PATH,
+    FILE_PUBLISHER_PATH,
+    REPO_PATH,
+)
+from pulp_smash.tests.pulp3.file.api_v3.utils import (
+    gen_importer,
+    gen_publisher,
+)
+from pulp_smash.tests.pulp3.file.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+from pulp_smash.tests.pulp3.pulpcore.utils import gen_repo
+from pulp_smash.tests.pulp3.utils import (
+    get_auth,
+    get_latest_repo_version,
+    read_repo_content,
+    sync_repo,
+)
+
+
+class AddingRemovingUnitsTestCase(unittest.TestCase, utils.SmokeTest):
+    """Create new repo version adding or removing content."""
+
+    def test_all(self):
+        """Create a new repo version adding or removing content.
+
+        This test explores the design choice stated in `Pulp #3234`_ that
+        create a new repository version by adding or removing content to the
+        latest version of the repository.
+
+        .. _Pulp #3234: https://pulp.plan.io/issues/3234
+
+        Do the following:
+
+        1. Add content to Pulp - using sync to a first repository.
```

Comment:
How about using a phrase like "create and sync a repository?" It doesn't matter whether we populate the repository by syncing in content, uploading content, copying content, or something else, so best not to mention it.